ui/raidboss/data/02-arr/raid/t1.ts

```diff
@@ -37,7 +44,7 @@ export default {
       netRegexCn: NetRegexes.ability({ source: '神杖巨蛇', id: '4BA' }),
       netRegexKo: NetRegexes.ability({ source: '카두케우스', id: '4BA' }),
       condition: Conditions.targetIsYou(),
-      alertText: (data, _, output) => output.text(),
+      alertText: (data: Data, _: unknown, output: Output) => output.text?.(),
```

Comment:
I wonder if, before going forward, it makes sense to make the change these `data, _, output` definitions to be `_data, _matches, output`, etc. now that we've added the rule in https://github.com/quisquous/cactbot/pull/2790

Reply:
Yeah, that seems like a reasonable change!

Reply:
Thanks for fixing it!